weaviate/warnings.py

```diff
@@ -115,3 +115,12 @@ def weaviate_too_old_vs_latest(server_version: str):
             category=DeprecationWarning,
             stacklevel=1,
         )
+
+    @staticmethod
+    def weaviate_client_too_old_vs_latest(client_version: str, latest_version: str):
+        warnings.warn(
+            message=f"""Dep004: You are using weaviate-client version {client_version}. The latest version is {latest_version}.
```

Comment:
Dep005

Reply:
Hi Dirk! Sure thing.

Also apologies for delay - also had vacation :)

I'll get these changes in this week.

Reply:
thank you very much!